changelogs/unreleased/gh-4726-execute-field-in-box.md

```diff
@@ -1,3 +1,3 @@
 ## bugfix/box
 
-* **[Breaking change]** Now `box.execute` cannot be called before `box.cfg` (gh-4726).
+* Breaking change Now `box.execute` cannot be called before `box.cfg` (gh-4726).
```

Comment:
???

Reply:
Fxd

Reply:
AFAIS, you reverted the change, but according to [those](https://www.notion.so/tarantool/box-drop-execute-field-from-uninitialized-box-a1214304a5c04227bda8e9bfea93721f) meeting notes we agreed that this change should be considered as non-breaking.